ui/lib/datatypes/tag.g.dart

```diff
@@ -24,6 +24,9 @@ class Tag {
   bool isUnifier;
   String unifierTagId;
   List<String> unifiesTagIds;
+  String group_id;
+  String group_name;
+  int group_index;
```

Comment:
I think we should stick with using the `groups` for now if possible so we can support if needed multiple groups. So we would only need to add:

```suggestion
  List<String> group_ids;
  List<int> group_indices;
```
(and if so, let's move them below the `List<String> groups` line)